ffi/src/post_impl.rs

```diff
@@ -62,6 +63,7 @@ fn _generate_proof(
     cfg: Config,
     nonces: usize,
     threads: usize,
+    pow_flags: post::pow::randomx::RandomXFlag,
```

Comment:
shared pow instance that is used in verifier can't be used for proving as well?

Reply:
It could be shared if it used the same flags. Most likely it won't as proving should use fast mode while verifying - light.

Reply:
thats understandable, but if fast is already used for proving i may use it for verifying as well

or your concern that it will require 2GB all the time, instead of requiring it only for short period of time?